src/Hid.Net/Windows/WindowsHidApiService.cs

```diff
@@ -153,14 +153,13 @@ private static string GetHidString(SafeFileHandle safeFileHandle, GetString getS
                 var pointerToBuffer = Marshal.AllocHGlobal(126);
                 var isSuccess = getString(safeFileHandle, pointerToBuffer, 126);
                 Marshal.FreeHGlobal(pointerToBuffer);
-
                 if (!isSuccess)
                 {
                     logger?.Log($"Could not get Hid string. Caller: {callMemberName}", nameof(WindowsHidApiService), null, LogLevel.Warning);
                 }
+                var text = Marshal.PtrToStringAuto(pointerToBuffer);
```

Comment:
Hi. It doesn't seem like there is any code change here. What bug is this change actually trying to fix? Could you please document an issue so that I understand what the issue is?